### DISCUSSION TITLE
Name squatting: MeshCentral

### DISCUSSION THREAD

**Discussion by NiceGuyIT** [community/community#65585] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

According to the GitHub's [Name Squatting Policy](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy#name-squatting-policy):

> GitHub prohibits account name squatting, and account names may not be reserved or inactively held for future use. Accounts violating this name squatting policy may be removed or renamed without notice.

The `MeshCentral`(1) name was created on 2020-05-01 and has no _public_ activity. I don't know if it's an org or username. There also is no contact information. Can you look to see if this is an org or personal account? Is there any way to contact them? Is the account active?

The reason is the MeshCentral personal repository ([Ylianst/MeshCentral](https://github.com/Ylianst/MeshCentral/)) might be converted to an org. The alternative is to use another name like `MeshCentral-community` or `MeshCentral-org`.

Note: I read some posts ([this in particular](https://github.com/orgs/community/discussions/23495)) that states GitHub does not review requests for dormant accounts. I don't know if that applies since I'm not asking to remove the account.

(1): `https://github.com/MeshCentral`

**Comment by RealJapaneseGreenTea** [comment]
Are you saying you want MeshCentral as your user ID?

**Reply by NiceGuyIT** [reply]
Not me. And not for a personal account. It would be an organization for the MeshCentral community. I'm just trying to see if it's possible and what is needed.

From [this post](https://github.com/orgs/community/discussions/28991), it looks like it might be possible because the only (public?) activity on the account is the creation of the account. I presume this violates the GitHub Name Squatting Policy but don't know. Hence the post.

**Comment by si458** [comment]
i would like `meshcentral` to be converted into an organization so i can add community members into it for the meshcentral project
im happy for it to be take it over as an organization, and i will assign @silversword411 as a contributor as well as myself being the admin
once we find someone flunent in C language they will get added too for the meshagent!
@Ylianst has agreed in an email that someone can take over at some point, but he just sadly hasnt had time to do it

**Comment by NiceGuyIT** [comment]
Bump.

**Comment by silversword411** [comment]
Checking <https://support.github.com/request>
Using _Account or billing issue_

They no longer deal with username squatting

<img width="519" alt="2023-09-05_143933 - brave_Accounts_-_GitHub_Support_-_Brave" src="https://github.com/community/community/assets/13395348/3ea40f83-66da-444f-9fe5-96dc3647739f">

They only deal with trademark squatting

<img width="535" alt="2023-09-05_143955 - brave_Accounts_-_GitHub_Support_-_Brave" src="https://github.com/community/community/assets/13395348/8ea12ad0-9ed6-476c-9609-79f5d5207213">

At this point, not being a Trademark holder...I don't think I can do anything further. Probably only Ylianst is the only one that could move this forward.

**Comment by terabytesoftw** [comment]
It's a shame that github can't solve the problem of people reserving names without any activity, just for the interest of selling the username, which is prohibited by github, let's hope they could have a proper naming policy, where this type of things can be solved.

**Reply by silversword411** [reply]
They have a policy, they just don't want the support or legal headache of enforcing it.
